content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -282,6 +289,141 @@ and [external.metrics.k8s.io](https://github.com/kubernetes/community/blob/maste
 For examples of how to use them see [the walkthrough for using custom metrics](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics)
 and [the walkthrough for using external metrics](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-metrics-not-related-to-kubernetes-objects).
 
+## Support for configurable scaling behavior
+
+Starting from
+[v1.17](https://github.com/kubernetes/enhancements/blob/master/keps/sig-autoscaling/20190307-configurable-scale-velocity-for-hpa.md)
+the `v2beta2` API allows scaling behavior to be configured through the HPA
+`behavior` field. Behaviors are specified separately for scaling up and down in
+`scaleUp` or `scaleDown` section under the `behavior` field. A stabilization
+window can be specified for both directions which prevents the flapping of the
+number of the replicas in the scaling target. Similarly specifing scaling
+policies controls the rate of change of replicas while scaling.
+
+### Scaling Policies
+
+One or more scaling policies can be specified in the `behavior` section of the spec.
+When multiple policies are specified the policy which allows the highest amount of
+change is the policy which is selected. The following example shows this behavior
+while scaling down:
```

Comment:
I'd suggest we move the 2nd sentence to somewhere before line 327, where we are discussing multiple policies and the selection policy.

Reply:
I would like to keep it because it's not clear what happens when multiple policies are specified. I will add some more context that it's the default behavior.